test/functional/cmdLineTests/jvmtitests/playlist.xml

```diff
@@ -460,4 +460,36 @@
 			<group>functional</group>
 		</groups>
 	</test>
-</playlist>
\ No newline at end of file
+	<test>
```

Comment:
Can this not be done in the standard JVMTI test framework?

Reply:
Existing `JVMTI` test suite runs on following modes https://github.com/eclipse/openj9/blob/ba27f519d48a2a4b6dd811212ec106313b734052/test/functional/cmdLineTests/jvmtitests/playlist.xml#L28-L38
in which `Mode3xx` and `Mode5xx` apply `-Xgcpolicy:metronome` and `-Xgcpolicy:balanced`. However these two modes are not supported by `J9HOOK_MM_OBJECT_ALLOCATION_SAMPLING` implemented via `mgcalloc.cpp:traceObjectCheck()` and `extensions->oolObjectSamplingBytesGranularity`.
So this new `soae001` test can't run as part of `JVMTI` suite in such modes. `jvmtitests_soae.xml` was created to run `cmdLineTester_jvmtitests_soae` with `Mode 110/148/149/610` (`Mode201 -Xrealtime` was removed in this testcase as well).

Reply:
OK